app/src/main/java/com/example/roomies/SettingsFragment.java

```diff
@@ -22,6 +22,7 @@
 import com.example.roomies.model.UserCircle;
 import com.parse.FindCallback;
```

Comment:
I cannot find a tests for this commits. Can you paste the testing link/plan here?

Reply:
https://github.com/srnazng/Roomies/commit/2ae138bce5ae18b073367d6eaaa876836fc07fe4#commitcomment-76669789